proofread:aws.md (#1980)
juicedata/juicefs: docs/en/tutorials/aws.md

JuiceFS clients will occupy 1GB of disk as cache by default. When dealing with a large number of files, the client will cache the data on disk first and then upload it to an object storage asynchronously. Choosing a disk with higher IO and reserving and setting a larger cache will allow JuiceFS to have better performance.

### 2. Object Storage

Amazon S3 is the de facto standard for public cloud object storage services, and the object storage services provided by other major cloud platforms are generally compatible with the S3 API. Thus, applications developed for S3 can switch object storage services between different platforms freely.

JuiceFS fully supports Amazon S3 and all S3-like object storage services, and you can see the documentation for [all object storage services supported by JuiceFS](../reference/how_to_setup_object_storage.md).

Amazon S3 offers a range of storage classes for different use cases, mainly including:

- Amazon S3 Standard: general-purpose storage for frequently accessed data

> - Amazon S3 Standard_IA: for data that is accessed less frequently and needs long-term storage 

Author: ```suggestion
- Amazon S3 Standard_IA: for data that is long-term stored but infrequently accessed
```

- Amazon S3 Glacier: for archiving data for long term

> Amazon S3 Standard classes is usually recommended for using JuiceFS because other classes may cost additional fee when retrieving data.

Reviewer: 哦那我理解错了，这里说的应该是 storageclass

2021/10/14 08:47:50.132003 juicefs[10601] <INFO>: OK, mystor is ready at /mnt/jfs
```

To check how the file system is mounted, you can use the command `df`, 

```shell
$ df -Th
File system      type             capacity used usable used%  mount point
JuiceFS:mystor   fuse.juicefs      1.0P    64K  1.0P    1%    /mnt/jfs
```

Once mounted, the file system can be used like a local disk, and the data stored in the directory `/mnt/jfs` is coordinated by the JuiceFS client and eventually stored in the S3 object storage.

> > **Multi-Host Sharing**: JuiceFS supports being mounted by multiple hosts at the same time, you can install the JuiceFS client on any cloud server on any platform. The files can be shared to read and write once mounting the file system using the database address `redis://:<your-redis-password>@herald-sh-abc.redis.rds.aliyuncs.com:6379/1`, but you need to make sure that the host on which the file system is mounted has proper access to the database and the S3 being used.

Reviewer: ```suggestion
> **Multi-Host Sharing**: JuiceFS supports being mounted by multiple hosts at the same time, you can install the JuiceFS client on any cloud server on any platform. The files can be shared to read and write once mounting the file system using the database address `redis://:<your-redis-password>@herald-sh-abc.redis.rds.aliyuncs.com:6379/1`, but you need to make sure that the host which the file system is mounted on has privileges to access the database and the S3 being used.
```
